src/api/app/controllers/labels/projects_controller.rb

```diff
@@ -1,21 +1,20 @@
 class Labels::ProjectsController < ApplicationController
   before_action :set_project
+  before_action :authorize_with_label_global_policy
```

Comment:
If there is no other authorization hook I would rename this to simply `before_action :authorize`

Reply:
I prefer to make a distinction here and leave it with a different name.